components/scream/src/physics/share/physics_functions.hpp

```diff
@@ -112,18 +112,30 @@ struct Functions
   // The result is the potential temperature, units in K
   // The inputs are
   //   T_mid is the atmospheric temperature, units in K
-  //   Exners function, unitless.  Exners function can be derived using `get_exner` defined above.
+  //   p_mid is the atmospheric pressure, units in Pa.  Used with Exners function using `get_exner` defined above.
   KOKKOS_FUNCTION
-  static Spack T_to_th(const Spack& T_atm, const Spack& exner, const Smask& range_mask);
+  static Spack get_potential_temperature(const Spack& T_mid, const Spack& p_mid, const Smask& range_mask);
```

Comment:
For what is worth, I thought `T_to_th` was fine. Maybe something a tad more verbose, like `T_to_Theta`.

Reply:
I'm happy to change it back if there is consensus on the names.  I know that `theta` is the typical term for potential temperature but I thought it was a bit ambiguous in the context of a function name.  Hence my decision to explicitly state potential temperature.  Is there any opposition to an even more verbose name like `get_potential_temp_from_temp`?  Again, totally happy to keep `T_to_Theta` if that is likely to be clear to users.  @PeterCaldwell , any thoughts?

Reply:
We can keep `get_potential_temperature`.

Reply:
Oh jeez, asking Aaron "Verbose" Donahue to name functions could be dangerous. ;-). I like long names only to the extend that we need them to be self-describing. Another goal I have is to name things in a consistent way. So having `get_potential_temperature` to convert T to theta but `Th2T` to convert from theta to T would make me cry. I think `get_potential_temperature` is a fine name. `T_to_Th` is too brief and `get_potential_temp_from_temp` is too long.